index.js

```diff
@@ -753,28 +753,40 @@ function Device(atemIpAddress){
 	});
 
 	this.on('PrvI', function(d) {
-		const source = d.readUInt16BE(2),
+		const me = d.readUInt8(0);
+		const source = d.readUInt16BE(2)
 			inTransition = (d[4] & 1) === 1;
 
 		/**
 		 * When the selected preview bus changes this event will be fired.
 		 * @event Device#previewBus
-		 * @property {SourceID} source The new preview source
+		 * @property {number} me 0 for M/E 1, 1 for M/E 2
+		 * @property {number} source The new preview source
+		 * @property {boolean} inTransition Is this currently dissolving?
 		 */
-		atem.emit('previewBus', source, inTransition);
```

Comment:
Changing the way the data is passed from separate arguments to an one argument object looks nice, but it breaks backwards compatibility though.
I'd suggest to keep the multiple arguments style for now and pass the ME information as a 3rd argument. This way we can release it as a sem-ver minor update. It is possible to release a major update afterwards where we switch to one argument objects but that is for another pull request...

Reply:
This is true, but you could also argue that since you're still in major version zero, the API isn't set in stone anyway.

That being said, I can also respect your desire to keep it stable even in version zero.  Also, since it's your repository, that's your call!  I'd be happy to change it to pass the ME information as a 3rd argument if you'd prefer.